packages/fxa-content-server/app/scripts/views/settings/display_name.jsx

```diff
@@ -0,0 +1,203 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import AvatarMixin from '../mixins/avatar-mixin';
+import Cocktail from 'cocktail';
+import BaseView from '../base';
+import SettingsPanelMixin from '../mixins/settings-panel-mixin';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Translator from '../../lib/translator';
+
+const translator = new Translator();
+
+const t = msg => translator.get(msg);
+
+function DisplayName(props) {
```

Comment:
Consider adding `export` to all these React components. While we do `export` the `View` at the end of the module, being able to access each individual component separately might make it easier to test things in smaller pieces.